core/src/main/java/org/mapfish/print/config/Configuration.java

```diff
@@ -449,17 +454,28 @@ public final void setDefaultStyle(final Map<String, Style> defaultStyle) {
     }
 
     /**
-     * If true then if the request JSON has extra parameters exceptions will be thrown.  Otherwise the
-     * information will be logged.
+     * If true and the request JSON has extra parameters in the layers definition, exceptions will be thrown.
+     * Otherwise the information will be logged.
      */
     public final boolean isThrowErrorOnExtraParameters() {
         return this.throwErrorOnExtraParameters;
     }
 
+    /**
+     * If true and the request JSON has extra parameters in the layers definition, exceptions will be thrown.
+     * Otherwise the information will be logged.
+     *
+     * @param throwErrorOnExtraParameters the value
+     */
     public final void setThrowErrorOnExtraParameters(final boolean throwErrorOnExtraParameters) {
         this.throwErrorOnExtraParameters = throwErrorOnExtraParameters;
     }
 
+    @Override
+    public void validate(final List<Throwable> validationErrors, final Configuration configuration) {
+        validationErrors.addAll(validate());
+    }
```

Comment:
What's this for?

Reply:
I need to implement the `ConfigurationObject` for the class to be included in the doc and this method is requested by this interface.

Reply:
OK, thanks :-)